.github/workflows/release.yml

```diff
@@ -0,0 +1,97 @@
+name: release
+
+on:
+  create:
+    tags: [ '[0-9]+.[0-9]+.[0-9]+.*' ]
+
+permissions:
+  contents: write
+
+env:
+  DOTNET_VERSION: '6.0.400'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+       fetch-depth: 0
```

Comment:
I believe we need `fetch-depth` of `1` to satisfy our `MinVer` tooling being able to determine the appropiate version number based on available git tags.

Reply:
See e.g https://github.com/elastic/elastic-ingest-dotnet/blob/main/.github/workflows/ci.yml#L46-L52

Reply:
IIUC, `fetch-depth: 0` retrieves everything 

```
    # Number of commits to fetch. 0 indicates all history for all branches and tags.
    # Default: 1
    fetch-depth: ''
```

as explained in https://github.com/actions/checkout

I guess there is no need to fetch anything if `0`

Reply:
Ahh perfect!